VisualizationBase/src/nodes/InfoNode.cpp

```diff
@@ -27,6 +27,9 @@
 #include "InfoNode.h"
 
 #include "ModelBase/src/nodes/TypedListDefinition.h"
+#include "QJsonArray"
```

Comment:
`precompiled.h`

In general, everytime you want to include a library header, it should go there.